docs/complex-transactions.md

```diff
@@ -7,7 +7,7 @@ description: How to build and send a complex Flow transaction from the command l
 **Simple Transactions**
 
 Sending a transaction using the Flow CLI can simply be 
-achieved by using the [send command documented here](/flow-cli/send-transactions/).
+achieved by using the [send command documented here](https://developers.flow.com/tools/flow-cli/send-transactions).
```

Comment:
shouldn't this be just `/tools/flow-cli...` without the domain?

Reply:
we can use /tools/flow-cli.. but I just add the whole link just to be sure

Reply:
we should use only / part

Reply:
addressed